package.json

```diff
@@ -33,7 +33,6 @@
     "@babel/core": "^7.13.10",
     "@typescript-eslint/eslint-plugin": "^4.0.0",
     "@typescript-eslint/parser": "^4.0.0",
-    "babel-eslint": "^10.0.1",
```

Comment:
This is unrelated but I just noticed that this was still here but we aren't using it